static/things.json

```diff
@@ -1,3 +0,0 @@
-[
-
-]
```

Comment:
LOL, what's that still doing there? Oh, I left it there 😂

Reply:
Haha I don't know how I've completely missed that file for over 3 years.